README.md

```diff
@@ -58,3 +58,22 @@ command:
 ```
 firefox com.ibm.streamsx.metrics/doc/spldoc/html/index.html &
 ```
+
+### Building Project
+The MetricsSink operator requires you to build jars and include them in your project. You will also need to have an ElasticSearch server running.
```

Comment:
To be clear, if the jars are built from the java files in the toolkit they should not be added to the source tree. Instead a release is produced which is a built version of the toolkit.

Ie. we should make the toolkit optimal for users that download a pre-built release of it.

I'm not sure what jars are being talked about.